src/service/video-manager-impl.js

```diff
@@ -441,6 +509,21 @@ class VideoEntry {
       },
     });
 
+    // Re-measure initial position when the window resizes / orientation changes
+    const viewport = viewportForDoc(this.ampdoc_);
+    viewport.onResize(() => {
+      this.vsync_.run({
+        measure: () => {
+          this.initialRect_ = this.video.element./*OK*/getBoundingClientRect();
+        },
+        mutate: () => {
+          this.dockingState_ = DockingStates.INLINE;
+          if (this.lastPosition_) {
+            this.onDockableVideoPositionChanged(this.lastPosition_);
```

Comment:
if you ended up using `getLayoutBox`, manager itself listens to viewport.onChange (wrongly, should have been onResize). You can move this there and have it call all entries `onDockableVideoPositionChanged` method if dockable. (same way it already does for visibility)